verification/domain/models/duns_verification.py

```diff
@@ -3,16 +3,17 @@
 
 from common.utils import datetime_to_string
 from verification.constants import DUNSVerificationStatus
+from verification.domain.models.comment import Comment
 
 
 class DUNSVerification:
-    def __init__(self, verification_id, org_uuid, status, comments, created_at, update_at):
+    def __init__(self, verification_id, org_uuid, status, comments, created_at, updated_at):
```

Comment:
We should add getter and setter, the same way as we have done for comment model.

Reply:
https://github.com/singnet/snet-marketplace-service/pull/1431